library/src/main/java/org/xmtp/android/library/SendOptions.kt

```diff
@@ -5,5 +5,6 @@ import org.xmtp.proto.message.contents.Content
 data class SendOptions(
     var compression: EncodedContentCompression? = null,
     var contentType: Content.ContentTypeId? = null,
-    var ephemeral: Boolean = false
+    var ephemeral: Boolean = false,
+    var __shouldPush: Boolean? = null
```

Comment:
why is `shouldPush` a send option? i thought the codecs are handling this value.

Reply:
@kele-leanes I agree on this we should be able to get if it shouldPush from the contentType which is part of the send options

Reply:
hi @rygine here is the [discussion](https://github.com/xmtp/xmtp-react-native/pull/235#discussion_r1484722744) where we came up with this approach